src/homework4/decorator.py

```diff
@@ -0,0 +1,20 @@
+"""
+Создайте декоратор, который хранит в себе результаты вызовов функций
+и время получения результата за время запуска программы.
+"""
+
+import time
+
+
+def log_results(func):
+    if not hasattr(log_results, "logs"):
+        log_results.logs = []
```

Comment:
why can't we use logs = []? 
overall it looks as copy of someone else solution (i don't remember exactly who wrote it but can find if needed)

Reply:
i will improve score if you can explain why do we need hasattr here

Reply:
i sent a message to python.epam.q42019@gmail.com